docs/examples/plot_linear_tree_tutorial.py

```diff
@@ -0,0 +1,76 @@
+"""
+Handling Data with Many Labels
+==============================
+
+Training a multi-label classification problem with numerous labels can be time-consuming. 
+We show how a solver in LibMultiLabel can reduce training time on such datasets.
+
+Consider the EUR-Lex dataset, which contains a large number of labels (3,956 labels) 
+and can be downloaded from `LIBSVM Data Sets <https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/multilabel.html>`_. 
+We show the use of the ``train_tree`` method and compare it with the standard one ``train_1vsrest``.
+To run the following code, you must put data to the designated directory.
+"""
+
+import libmultilabel.linear as linear
+import time
+
+preprocessor = linear.Preprocessor(data_format='txt')
+datasets = preprocessor.load_data('data/eur-lex/train.txt',
+                                  'data/eur-lex/test.txt')
+
+training_start = time.time()
+# the standard one-vs-rest way to train multi-label problems
+model_OVR = linear.train_1vsrest(datasets['train']['y'],
+                                 datasets['train']['x'],
+                                 '-s 2')
```

Comment:
no need to have -s 2 as we are not teaching people how to set parameters. The same for line 33